dwave/system/samplers/dwave_sampler.py

```diff
@@ -43,6 +90,19 @@ class DWaveSampler(dimod.Sampler, dimod.Structured):
     Inherits from :class:`dimod.Sampler` and :class:`dimod.Structured`.
 
     Args:
+        failover (bool, optional, default=False):
```

Comment:
This was already discussed in comments, but I would argue for default `True`. It makes the most sense, long term. Not sure how big is the problem of EC layer breaking, assuming most users will use AEC? You should weigh in, but I assert the (default) API should make most sense for most users.

Reply:
IMO preferred pattern (when possible) is to use the `FixedVariableComposite` or `LazyFixedEmbeddingComposite`. I also think that it would be a significant change to the "contract" of the API for it to change.